### PR TITLE
Implement kubermatic-installer deploy monitoring

### LINKED ISSUE #5860: Kubermatic Installer: Implement Monitoring Stack
**User Story**
The installer should be able to install the monitoring stack.

**Acceptance criteria**
- [ ] `./installer deploy monitoring` should be implemented.

### PR DESCRIPTION
**Pull request by mohamed-rafraf**
**What this PR does / why we need it**:
You can install the monitoring stack easily with `kubermatic-installer deploy monitoring`
**Which issue(s) this PR fixes**:

Fixes #5860

**What type of PR is this?**

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: cmd/kubermatic-installer/cmd_deploy.go [hunk 1]
```diff
@@ -95,7 +98,7 @@ func DeployCommand(logger *logrus.Logger, versions kubermaticversion.Versions) *
 	}
 
 	cmd := &cobra.Command{
-		Use:          "deploy [kubermatic-master | kubermatic-seed | usercluster-mla]",
+		Use:          "deploy [kubermatic-master | kubermatic-seed | usercluster-mla | monitoring]",
```
Comment (xrstf) [inline cmd/kubermatic-installer/cmd_deploy.go]: Please keep the 4 options sorted alphabetically

**Comment by xrstf** [comment]
So in general I wonder if it makes sense to have a "monitoring" stack. Currentely we offer a `usercluster-mla` stack in the installer, maybe it woul make sense to have a `seed-mla` stack that installs the charts from monitoring/* and logging/*. IMHO there is no need for the installer to map charts 1:1 by directory.

Also for the admin it might be difficult to discern what the difference between "monitoring" and "usercluster-mla" is ("isn't the M in MLA also monitoring?").

@embik WDYT?

**Comment by xrstf** [comment]
This is superseded by #13360, right?

**Comment by mohamed-rafraf** [comment]
> This is superseded by #13360, right?

Yes

**Pull request closed without merging**